common/src/main/java/org/apache/uniffle/common/rpc/GrpcServer.java

```diff
@@ -77,6 +78,11 @@ protected GrpcServer(
             ThreadUtils.getThreadFactory("Grpc"),
             grpcMetrics);
   }
+  
+  //This method is only used for testing
+  public static boolean getExecuted() {
```

Comment:
Because this is boolean type. We would better use `isPoolExecutorHasExecuted` instead of `getExecuted`.